server/src/main/java/org/opensearch/rest/action/admin/cluster/RestClusterStatsAction.java

```diff
@@ -64,10 +102,102 @@ public String getName() {
 
     @Override
     public RestChannelConsumer prepareRequest(final RestRequest request, final NodeClient client) throws IOException {
-        ClusterStatsRequest clusterStatsRequest = new ClusterStatsRequest().nodesIds(request.paramAsStringArray("nodeId", null));
+        ClusterStatsRequest clusterStatsRequest = fromRequest(request);
+        return channel -> client.admin().cluster().clusterStats(clusterStatsRequest, new NodesResponseRestListener<>(channel));
+    }
+
+    public static ClusterStatsRequest fromRequest(final RestRequest request) {
+        Set<String> metrics = Strings.tokenizeByCommaToSet(request.param("metric", "_all"));
+        String indicesMetricsDefaultValue = metrics.contains(Metric.INDICES.metricName()) || metrics.contains("_all") ? "_all" : null;
```

Comment:
can you add a code comment and as well to the documentaion that default filter is _all for indicesMetrics ?

Reply:
Added.